templates/ubuntu-frame.html

```diff
@@ -22,7 +22,16 @@ <h2>The foundation for embedded graphical display implementations</h2>
       <p>Ubuntu Frame streamlines the build and development of products that need graphical output. Ubuntu Frame bundles communication protocols, input protocols, and security policies into a single kit, which can then be used in embedded devices. Whether you are building an interactive kiosk, a digital signage solution or any other product that requires a graphical output, Ubuntu Frame instantly allows your device to run your fullscreen application.</p>
     </div>
     <div class="col-6 u-hide--small u-hide--medium u-align--center">
-      <img src="https://assets.ubuntu.com/v1/79e5a4c2-Ubuntu+Frame.svg" alt="Ubuntu frame" style="height: 48px; width: 358px;">
+      {{
+        image (
+        url="https://assets.ubuntu.com/v1/540688d1-Canonical-Ubuntu-Frame-logo-2022.svg",
+        alt="Ubuntu Frame trademark",
```

Comment:
```suggestion
        alt="",
```
I don't think this alt text is needed as the image doesn't provide anything extra to the section